doc/api.md

```diff
@@ -38,34 +34,18 @@ A user of the Cognoma system.
 | email | string | Optional email if the user wishes to get notifications | N |
 | last_login | datetime | Last time the user logged in. | Y |
 
-### Algorithm (/algorithms)
-
-Machine learning algorithms available in Cognoma.
-
-
-| Field        | Type           | Description | Read-Only
-| ------------- |:-------------:| ----------:| ----------:|
-| name | string | Primary Key. | Y |
-| parameters | object | JSON Schema used to generate algorithm parameter forms and displays. | Y |
-
 ### Gene (/genes)
 
 Reference table for genes within Cognoma. Entire model is read-only.
 
-
 | Field        | Type           | Description |
 | ------------- |:-------------:| ----------:|
-| entrezid | integer | Primary Key. |
-| systematic_name | string |  |
-| standard_name | string |  |
+| entrez_gene_id | integer | Primary Key. |
```

Comment:
[Opened an issue](https://bitbucket.org/greenelab/django-genes/issues/3/python-3-compatability) to check for Python 3 compatibility of `django-genes`.